components/youtube_script_injector/browser/content/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
+  "+content/public/browser",
+  "+third_party/blink/public",
+  "+brave/browser/youtube_script_injector",
+  "+chrome/common",
```

Comment:
this is a layering violation, you cannot use `chrome` inside `components`. Same with `brave/browser`